exonum-java-binding/core/rust/exonum-java/start_cryptocurrency_node.sh

```diff
@@ -45,9 +45,7 @@ echo "${SERVICE_NAME} = '${ARTIFACT_PATH}'" >> ${SERVICES_CONFIG_FILE}
 header "PREPARE PATHS"
 
 EJB_LOG_CONFIG_PATH="${EJB_APP_DIR}/log4j-fallback.xml"
-
-export LD_LIBRARY_PATH="$JAVA_LIB_DIR"
-echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}"
+JAVA_LIBRARY_PATH="${EJB_ROOT}/core/rust/target/debug"
```

Comment:
Is it always `debug` when cargo run is used (it’s probably fine for these scripts if it unconditionally is)?

Reply:
Yeap, `cargo run` without `--release` uses the `dev` profile that always works with `debug` artifacts.